conversation.pl

```diff
@@ -90,6 +106,39 @@
 dialog_state(grandma, 0, _) :-
 	println("Bye.").
 	
+	
+dialog_state(spider, 1, 0) :- 
+	\+ spider_gone,
+	println("There is a spider in a small glade."),
+	dialog_state(spider, 2, 0).
+
+dialog_state(spider, 2, 0) :-
+	println("0 - Leave slowly."),
+	println("1 - Look closer."),
+	println("2 - Attack."),
+	println("3 - Scare it away."),
+	read(Response),
+	dialog_state(spider, Response, 1).
+	
+dialog_state(spider, 1, 1) :-
+	println("It seems that the spider did not notice you yet."),
+	println("Despite the fact that it is indeed quite large, you have a feeling that it will not attack if not provoked."),
+	println("Small insects are often scared of people. You wonder if that is also the case for the bigger ones."),
+	dialog_state(spider, 2, 0).
+
+dialog_state(spider, 2, 1) :-
+	println("As you charge at the spider, the fight begins..."),
+	assertz(is_enemy(spider)).
+
+dialog_state(spider, 3, 1) :-
+	println("You sneak on a spider slowly. You manage to get close, breathe in, and then suddenly jump and scream as loud as you can."),
+	println("The spider freaks out and escapes."),
+	assertz(spider_gone),
```

Comment:
spider_gone jest już zdefiniowany jako regula:
spider_gone:-
	not(is_alive(spider)).
	
spider_gone:-
	not(position(spider, _)).
więc tutaj lepiej usunąć pozycję pająka.